content/methods/make/prototyping.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Prototyping
+permalink: /methods/make/prototyping/
+redirect_from:
+  - prototyping/
+layout: layouts/method-card
+tags: methods
+eleventyNavigation:
+  key: methods
+  parent: Make
+  title: Prototyping
+method:
+  title: Prototyping
+  what: A rudimentary version, either static or functional, of something that exhibits realistic form and function.
+  why: |
+    To enable direct examination of a design concept’s viability with a number of other methods such as <a href="https://methods.18f.gov/validate/usability-testing/#usability-testing" class="usa-link">usability testing</a> or a <a href="https://methods.18f.gov/discover/cognitive-walkthrough/#cognitive-walkthrough" class="usa-link">cognitive walkthrough</a>. Static prototypes (often paper) are helpful for gaining feedback on users’ intentions and various design elements. Functional prototypes (often coded) are helpful for observing how users interact with the product.
```

Comment:
@jduss4 [blocking request] I would just change these hard-coded links to `guides.18f.gov/methods/xyz` for now.

Reply:
Unfortunately won't that mean that they are 404s?

Reply:
For now, yes, but I'm concerned about forgetting them when they go live. They don't seem to be getting caught in the link tests.

Reply:
Okay, I'll switch them up!  I created an [issue yesterday](https://github.com/18F/guides/issues/298) to help us remember, but I also don't mind changing them now if it means less work later.